oauthdoctor/oauthdoctor.go

```diff
@@ -101,7 +102,11 @@ func main() {
 		log.Printf("Config file validation failed: %s\n", err)
 	}
 
-	cid := oauth.ReadCustomerID()
+	cid := strings.ReplaceAll(*customerID, "-", "")
```

Comment:
Can you please write a unit test to cover at least two cases?
- with given -customerid, make sure the stdin prompt not showing up
- without given -customerid, make sure the stdin prompt showing up

Reply:
Thanks for your comments. I'm newbie in Go-lang, so writing those tests is difficult. Also, I don't think testing is essential for simple argument processing that is not the main logic for the application. I am enjoying the forked application. Thank you very much again.

Reply:
I see. I think it is worth adding a test even though the logic is simple. The test cases are also relatively simply. Understanding your background, I can add the tests instead. Thank you for your idea and contribution!

Reply:
We really need to have a unit test for this. I appreciate that you are new to Go, but I would encourage you to look at the unit tests that we have written in this program. 

The book 'The Go Programming Language' by Donovan and Kernighan has an excellent description of testing in Go in Chapter 11. 

Thank you for your open source contribution. We will be glad to review your unit test when you submit it.